samples/drivers/led_apa102c_bitbang/src/main.c

```diff
@@ -79,12 +79,12 @@ void main(void)
 	}
 
 	/* Setup GPIO output */
-	ret = gpio_pin_configure(gpio_dev, GPIO_DATA_PIN, (GPIO_DIR_OUT));
+	ret = gpio_pin_configure(gpio_dev, GPIO_DATA_PIN, (GPIO_OUTPUT));
```

Comment:
There's really no reason why that parameter needs to be in parentheses, but changing it isn't necessary.

Reply:
Fixed, thanks